pkg/sensors/base/base.go

```diff
@@ -68,6 +68,8 @@ var (
 	StatsMap           = program.MapBuilder("tg_stats_map", Execve)
 
 	MatchBinariesSetMap = program.MapBuilder(mbset.MapName, Execve)
+
+	ErrMetricsMap = program.MapBuilder("errmetrics_map", Execve)
```

Comment:
should we use the `tg` prefix for newer maps?

Reply:
sure, fixed.